packages/@aws-cdk-testing/cli-integ/resources/cdk-apps/rollback-test-app/app.js

```diff
@@ -0,0 +1,99 @@
+const cdk = require('aws-cdk-lib');
+const lambda = require('aws-cdk-lib/aws-lambda');
+const cr = require('aws-cdk-lib/custom-resources');
+
+/**
+ * This stack will be deployed in multiple phases, to achieve a very specific effect
+ *
+ * It contains resources r1 and r2, where r1 gets deployed first.
+ *
+ * - PHASE = 1: both resources deploy regularly.
+ * - PHASE = 2: r1 gets updated, r2 will fail to update, and r1 will fail its rollback.
```

Comment:
Needs to be adjusted for phases `2a` and `2b`. I'm good with just removing this and let the tests speak for themselves.